### PR TITLE
Add proxmox example config

### PR DESCRIPTION
**Pull request by lkmhaqer**
This is my attempt at the proxmox configuration example, I think it's a bit more opinionated than my previous examples. Open to feedback on all aspects of the document. Particularly unsure if the ACL section is a little too verbose, or possibly overlaps with Proxmox's own documentation too much.

### CODE REVIEW

File: example_configs/proxmox.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- In this menu, select `Add > LDAP Server`
+
+## General Options
+
+- Realm: The internal proxmox name for this authentication method
+- Base Domain Name: `dc=example,dc=com`
+- User Attribute Name: `uid`
+- Server: Your LLDAP hostname or IP
+- Port: `3890`
+- SSL: Leave unchecked unless you're using SSL
```
Comment (nitnelave) [inline example_configs/proxmox.md:18]: LDAPS maybe?

File: example_configs/proxmox.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+## Sync Options
+
+- Bind User: `uid=admin,ou=people,dc=example,dc=com`
+  - It is recommended that you create a separate user account (e.g, `bind_user`) instead of `admin` for sharing Bind credentials with other services. The `bind_user` should be a member of the `lldap_strict_readonly` group to limit access to your LDAP configuration in LLDAP.
+- Bind Password: password of the user specified above
+- E-Mail Attribute: `mail`
+- Groupname attr: `cn`
+- User Filter: `(&(objectClass=person)(|(memberof=cn=proxmox_user,ou=groups,dc=example,dc=com)(memberof=cn=proxmox_admin,ou=groups,dc=example,dc=com)))`
+  - This filter will only copy users that are members of the `proxmox_user` or `proxmox_admin` groups. If you want to enable all users in lldap, this filter can be used: `(objectClass=person)`
+- Group Filter: `(&(objectClass=groupofnames)(|(cn=proxmox_user)(cn=proxmox_admin)))`
```
Comment (nitnelave) [inline example_configs/proxmox.md:30]: Nit: group of unique names

**Review by nitnelave — APPROVED** [review]
Nice! Thanks

**Comment by nitnelave** [comment]
Hey, do you want to update this PR, or should I take it over?

**Comment by lkmhaqer** [comment]
Ah sorry meant to get this updated before I went on Holiday. I should be
back tomorrow and I can clean this up.

/Charles

**Comment by lkmhaqer** [comment]
Updated PR per the review comments, let me know if this looks okay?

**Review by nitnelave — APPROVED** [review]
Thanks! A note for the future, though: it's syncHronization, not syncronization ;)

**Pull request merged**